src/Chapter2.hs

```diff
@@ -841,7 +875,8 @@ list.
 
 🕯 HINT: Use the 'cycle' function
 -}
-rotate = error "rotate: Not implemented!"
+rotate :: Int -> [Int] -> [Int]
+rotate x y = drop x $ take (x+ length y) $ cycle y 
```

Comment:
I think this function doesn't return empty list on negative rotations :thinking: 

Also, `cycle` fails in runtime on empty lists, so you need to handle this case separately.